modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceChannels.macro

```diff
@@ -76,6 +76,56 @@ definition {
 		Button.clickSave();
 	}
 
+	macro addFixedTaxRate {
+		var fieldLabel = "Amount";
+
+		Click(
+			key_entryName = "Fixed Tax Rate",
+			locator1 = "CommerceEntry#CHANNELS_GENERAL_TAB_EDITS");
+
+		SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+
+		Check.checkToggleSwitch(
+			key_toggleSwitchLabel = "Active",
+			locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+		Button.clickSave();
+
+		if (isSet(percentage)) {
+			Check.checkToggleSwitch(
+				key_toggleSwitchLabel = "Percentage",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+			Button.clickSave();
+
+			var fieldLabel = "Rate";
+		}
```

Comment:
Please make this whole macro a generic macro for the edit of a tax calculation, as all existing tax calculations in a channel share the same Details page.
Then, in the macro, based on the tax calculation passed(By Address, Remote, Fixed Tax Rate), the macro will call a different macro that manages the Tax Rates/Settings for each tax calculation.